### PR TITLE
Expose some buffer related functions

### PR DESCRIPTION
**Pull request by doronbehar**
Hi!

I really enjoy this plugin. I just wrote a customization to the `:Buffers` command I shared at https://github.com/junegunn/fzf/wiki/Examples-(vim)#delete-buffers-from-buffers and this requires these small changes. I hope it'll be OK with you.

Regards.

### CODE REVIEW

**Comment by junegunn** [comment]
See my previous comment: https://github.com/junegunn/fzf.vim/pull/1024#issuecomment-627323510

Making things public means that I'll have to pay extra care not to break backward compatibility from then on, and I don't want to do it and I want to make it clear by naming these functions with `_` prefix.

**Comment by doronbehar** [comment]
Done :guitar:.

File: autoload/fzf/vim.vim [hunk 1]
```diff
@@ -640,15 +640,15 @@ function! s:sort_buffers(...)
   return b1 < b2 ? 1 : -1
 endfunction
 
-function! s:buflisted_sorted()
+function! fzf#vim#_buflisted_sorted()
```
Comment (junegunn) [inline autoload/fzf/vim.vim:643]: There is still `s:buflisted_sorted` in the code.

Comment (doronbehar) [reply autoload/fzf/vim.vim:643]: :+1: I think I catched them all (with git grep).

**Comment by yanzhang0219** [comment]
Hi @doronbehar 

I encounter an error when I run the customized version of `:Buffers` command and try to delete a buffer using `D`. I attached the screenshot below. Thank you.

image

**Comment by doronbehar** [comment]
Yea I know, another person has contacted me personally to note this fault. However, from my experience these errors are not consistent - they don't appear always and they are not always reproducible, at least for me.

The problem is, that when `D` is pressed, we want to change something both in "the parent" Nvim session and in the list of open buffers in fzf "the child". This makes it hard to decide whether `D` should be mapped to something by an `autocmd` (By Vim) - which will make it easy to delete buffers with it, or to use `fzf`'s `--bind`. In the current implementation, the latter choice was made.

I'm currently unavailable to solve this little quirk, though it's in my TODO now. I will find the time probably at the end of July, when my semester will over.

P.S, sorry `@junegunn` for abusing your repo to discuss this.

**Comment by yanzhang0219** [comment]
Okay. I think it is better to give a tag/mark like "bug" to this customized command in the wiki page so that other users could recognize it and wait for it being solved. Thanks.

**Comment by doronbehar** [comment]
I've removed the wiki section as I've reached the conclusion it'll be impossible to implement this idea without the feature described in https://github.com/junegunn/fzf/issues/2019 .

**Pull request merged**